dev-util/coverity/coverity-2017.07.ebuild

```diff
@@ -0,0 +1,92 @@
+# Copyright 1999-2018 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
+
+DESCRIPTION="Static code analysis and dynamic code analysis tools"
+HOMEPAGE="https://www.coverity.com"
+SRC_URI="amd64? ( cov-analysis-linux64-${PV}.tar.gz )
+	x86? ( cov-analysis-linux-${PV}.tar.gz )"
+
+LICENSE="coverity-PLA"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="doc"
+
+L10N="ja ko zh-CN"
+for _l in ${L10N} ; do
+	IUSE="${IUSE} l10n_${_l}"
+done
+
+RESTRICT="fetch mirror bindist"
+
+QA_PREBUILT="opt/coverity/bin/*"
+
+pkg_nofetch() {
+	eerror "Please go to https://scan.coverity.com/download"
+	use amd64 && eerror "and download Linux64"
+	use x86   && eerror "and download Linux32"
+	eerror "then put the file in DISTDIR"
+	use amd64 && eerror "named as cov-analysis-linux64-${PV}.tar.gz"
+	use x86   && eerror "named as cov-analysis-linux-${PV}.tar.gz"
+}
+
+src_unpack() {
+	default
+
+	use amd64 && mv "${WORKDIR}/cov-analysis-linux64-${PV}" "${S}"
+	use x86   && mv "${WORKDIR}/cov-analysis-linux-${PV}" "${S}"
+}
+
+src_prepare () {
+	default
+
+	# Rename invalid locale files
+	mv locale/zh-cn locale/zh-CN || die "Failed to rename locale"
+
+	# Cleanup
+	find -type f -name '*.DS_Store' -exec rm {} + || die
```

Comment:
`-delete` should be equivalent here, or am I missing something?

Reply:
Indeed, that's leftover from early stage